src/router/routes/live/events.ts

```diff
@@ -110,7 +110,7 @@ export class EventsController extends LiveController {
     response: express.Response,
     next: express.NextFunction,
   ) {
-    if (!request.body || !request.body.event) {
+    if (!request.body || !request.body) {
```

Comment:
Leave this the same. Instead, change the line below to `new Event(request.body.event)`